src/indexable.cr

```diff
@@ -281,6 +281,17 @@ module Indexable(T)
     self
   end
 
+  # Returns an `Array` with all the elements in the collection.
+  #
+  # ```
+  # { 1, 2, 3 }.to_a # => [1, 2, 3]
```

Comment:
Please run formatting (`crystal tool format`) also for the example code:

```crystal
{1, 2, 3}.to_a # => [1, 2, 3]
```

Thank you.